ports/nrf/boards/WRL13990/pins.csv

```diff
@@ -0,0 +1,52 @@
+P2,P2
+P3,P3
+P4,P4
+P5,P5
+P6,P6
+P7,P7
+P8,P8
+P9,P9
+P10,P10
+P11,P11
+P12,P12
+P13,P13
+P14,P14
+P15,P15
+P16,P16
+P17,P17
+P18,P18
+P19,P19
+P20,P20
+P21,P21
+P22,P22
+P23,P23
+P24,P24
+P25,P25
+P26,P26
+P27,P27
+P28,P28
+P29,P29
+P30,P30
+P31,P31
+BTN,P6
```

Comment:
Could it be an option to replace the template definitions on top with the new functionality names and get them listed in ascending order? 

Also, if there is a need for fixing the position of SPI1 it would be good to align the names between SPI0 and SPI1, CLK and SCK.

Reply:
I would like to keep the generic names P0..Pxx as well as the labeled pins. Maybe just move all the labeled pins up (first to see) and the rest of the list to follow. Having both generic and labeled names in the list is a pattern that is also used in the STM32 boards csv files.

So with the STM32 in mind I would leave the pin order as is.

Reply:
If i recall correctly the duplicate definition would not have any effect. The make-pins.py for nrf-port is a bit different compared to the stm32 port.

The expectation is that, 1. column will generate the name for Pin.board, and the 2. column for Pin.cpu. This means that a definition of 'BTN,P6' should render Pin.board.BTN and Pin.cpu.P6. This will keep both the generic name from the CPU and the unique name for the Board.

What happens for the duplicate is a bit unknown. Does it run on target, can you do the lookup properly?